2014-05-14T11:27:50.735000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\addhost_sdnsplus' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.532000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\analyzeMFT' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.561000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\appdotnet' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.588000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\autorun' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.646000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\bottle' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.693000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\box' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.738000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\box.py' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:55.750000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\boxpython' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.047000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\cddvdlist' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.101000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\checkconn' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.150000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\CheesePrism' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.167000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\close' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.192000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\cpa' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.257000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\cservice' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.311000 - MOVE HG Dir : 'd:\PROJECT_EDIT\cython-plugin' --> 'd:\PROJECT_EDIT\hg'
2014-05-14T11:27:56.346000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\daemoncmd' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.472000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\dbsearch' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.533000 - MOVE HG Dir : 'd:\PROJECT_EDIT\DBSearch_Project' --> 'd:\PROJECT_EDIT\hg'
2014-05-14T11:27:56.584000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\DiffMove' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.608000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\DiffMovetest' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:56.633000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\django-extensions' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.218000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\fondu' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.257000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\gitmaster' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.312000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\GpsTracker' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.684000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\havarti' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.931000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\jpype' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.960000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\jso' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:57.973000 - MOVE HG Dir : 'd:\PROJECT_EDIT\kiwisyslog_web' --> 'd:\PROJECT_EDIT\hg'
2014-05-14T11:27:58.004000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\leo-editor' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.022000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\maillib' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.062000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\markup' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.218000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\multicastserver' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.234000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\mva' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.270000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\mybudget' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.285000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\mypi' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.378000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\o2' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.389000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\oauth2' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.439000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\oauth2-ruby' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.553000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pdf.js' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:58.910000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\php-oauth-client' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:59.092000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pip' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:27:59.180000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pyboxapi' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:01.839000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pyhelp' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:01.870000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pyoauth2' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:01.955000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pyramid' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:02.067000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\pyscheduler' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.047000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 221, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 5] Access is denied: \'d:\\\\PROJECT_EDIT\\\\pyscheduler\\\\.git\\\\objects\\\\00\\\\3835275bb0b3b55b85433160dfbc4bdd215d63\'\n')
2014-05-14T11:28:11.267000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\python-boxdotcom' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.300000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\python-oauth2' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.375000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\rack-oauth2' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.422000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\rauth' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.467000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\rn' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.523000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\ruby-oauth2' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.597000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\sanction' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.658000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\scribe' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.923000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\setuptools-git' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:11.994000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\simplepypi' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:12.209000 - MOVE HG Dir : 'd:\PROJECT_EDIT\syslog_center_server' --> 'd:\PROJECT_EDIT\hg'
2014-05-14T11:28:14.213000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\tornado' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:14.324000 - MOVE HG Dir : 'd:\PROJECT_EDIT\TotalcmdLog' --> 'd:\PROJECT_EDIT\hg'
2014-05-14T11:28:14.444000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\UniUpload' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:14.567000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\vhostmaker' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:14.675000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\wensleydale' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:28:14.714000 - MOVE GIT Dir: 'd:\PROJECT_EDIT\www.dotnetopenauth.net' --> 'd:\PROJECT_EDIT\git'
2014-05-14T11:50:17.527000 - MOVE GIT Dir: 'd:\PROJECT\banshee' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:50:35.585000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\banshee\\\\.git\\\\objects\\\\pack\\\\pack-fb26984e68c029247461f16b1a835754a01f7e99.idx\'\n')
2014-05-14T11:50:35.667000 - MOVE GIT Dir: 'd:\PROJECT\BCK\git\banshee' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:50:35.671000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\banshee\' already exists\n')
2014-05-14T11:50:35.917000 - MOVE GIT Dir: 'd:\PROJECT\cabal' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:50:40.608000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\cabal\\\\.git\\\\objects\\\\pack\\\\pack-f5eecf5031c640798af4d0b34c61d92bfba738e5.idx\'\n')
2014-05-14T11:50:40.687000 - MOVE GIT Dir: 'd:\PROJECT\carikost.git' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:50:46.284000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\carikost.git\\\\.git\\\\objects\\\\pack\\\\pack-95b2083a3e445fe33344577861bf32a9b216d210.idx\'\n')
2014-05-14T11:50:46.567000 - MOVE GIT Dir: 'd:\PROJECT\cddvdlist' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:51:53.541000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\cddvdlist\\\\.git\\\\objects\\\\pack\\\\pack-a6dd7be3d83cecfe0ba13d1107b6db21d03050e4.idx\'\n')
2014-05-14T11:51:53.720000 - MOVE GIT Dir: 'd:\PROJECT\clive' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:51:54.031000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\clive\\\\.git\\\\objects\\\\pack\\\\pack-85de16b4735bcbc552e452bdaaebf22e14c348bf.idx\'\n')
2014-05-14T11:51:54.081000 - MOVE GIT Dir: 'd:\PROJECT\connect-js' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:51:54.246000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\connect-js\\\\.git\\\\objects\\\\pack\\\\pack-fd418080e57b68303459f1659bd47448560e449a.idx\'\n')
2014-05-14T11:51:54.844000 - MOVE GIT Dir: 'd:\PROJECT\dulwich' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:00.422000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\dulwich\\\\.git\\\\objects\\\\pack\\\\pack-8ccf26ece9b925ee0ed9afd0d87e7d696e19963a.idx\'\n')
2014-05-14T11:52:00.520000 - MOVE GIT Dir: 'd:\PROJECT\Facebook.Python.SDK' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:00.590000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\Facebook.Python.SDK\\\\.git\\\\objects\\\\pack\\\\pack-f6b293d38197aa64b030cce90f9ed2a44c8acc24.idx\'\n')
2014-05-14T11:52:00.601000 - MOVE GIT Dir: 'd:\PROJECT\fast-export' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:00.753000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\fast-export\\\\.git\\\\objects\\\\pack\\\\pack-d8c7bd8f0a57bee6f1f7b8001dd23d396f119887.idx\'\n')
2014-05-14T11:52:00.758000 - MOVE GIT Dir: 'd:\PROJECT\FeedNotifier' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:00.947000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\FeedNotifier\\\\.git\\\\objects\\\\pack\\\\pack-156f9f3152a18444842f9987b1522b0e6419e8b2.idx\'\n')
2014-05-14T11:52:01.008000 - MOVE GIT Dir: 'd:\PROJECT\five-android' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:04.740000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\five-android\\\\.git\\\\objects\\\\pack\\\\pack-c9eba17b770575a4f790cfba7cd32086d8ee6029.idx\'\n')
2014-05-14T11:52:04.789000 - MOVE GIT Dir: 'd:\PROJECT\five-server' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:08.466000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 5] Access is denied: \'d:\\\\PROJECT\\\\five-server\\\\.git\\\\objects\\\\0b\\\\fc9540cebb8762a240a70964eaf4065e885b2f\'\n')
2014-05-14T11:52:08.596000 - MOVE GIT Dir: 'd:\PROJECT\ftpparsemodule' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:08.838000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\ftpparsemodule\\\\.git\\\\objects\\\\pack\\\\pack-5337ef0e18ea0e0868ab422546a43611781461f1.idx\'\n')
2014-05-14T11:52:08.845000 - MOVE GIT Dir: 'd:\PROJECT\girocco' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:09.002000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 299, in move\n    rmtree(src)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 247, in rmtree\n    rmtree(fullname, ignore_errors, onerror)\n  File "C:\\Python27\\lib\\shutil.py", line 252, in rmtree\n    onerror(os.remove, fullname, sys.exc_info())\n  File "C:\\Python27\\lib\\shutil.py", line 250, in rmtree\n    os.remove(fullname)\nWindowsError: [Error 32] The process cannot access the file because it is being used by another process: \'d:\\\\PROJECT\\\\girocco\\\\.git\\\\objects\\\\pack\\\\pack-9ef428217a1e548334faebb67eaee2692d58364e.idx\'\n')
2014-05-14T11:52:10.854000 - MOVE GIT Dir: 'd:\PROJECT\git\adfly-proxy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:10.872000 - MOVE GIT Dir: 'd:\PROJECT\git\apachelog-1.0' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:10.902000 - MOVE GIT Dir: 'd:\PROJECT\git\apt-cyg' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:10.977000 - MOVE GIT Dir: 'd:\PROJECT\git\banshee' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:11.001000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\banshee\' already exists\n')
2014-05-14T11:52:14.223000 - MOVE GIT Dir: 'd:\PROJECT\git\cabal' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:14.229000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\cabal\' already exists\n')
2014-05-14T11:52:14.824000 - MOVE GIT Dir: 'd:\PROJECT\git\carikost.git' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:14.829000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\carikost.git\' already exists\n')
2014-05-14T11:52:17.002000 - MOVE GIT Dir: 'd:\PROJECT\git\cddvdlist' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:17.006000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\cddvdlist\' already exists\n')
2014-05-14T11:52:17.356000 - MOVE HG Dir : 'd:\PROJECT\git\cddvdlist\MyFilebank_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:17.392000 - MOVE GIT Dir: 'd:\PROJECT\git\clive' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:17.396000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\clive\' already exists\n')
2014-05-14T11:52:17.601000 - MOVE GIT Dir: 'd:\PROJECT\git\cmdow_v' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:17.631000 - MOVE GIT Dir: 'd:\PROJECT\git\connect-js' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:17.636000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\connect-js\' already exists\n')
2014-05-14T11:52:17.810000 - MOVE GIT Dir: 'd:\PROJECT\git\CPM' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:17.882000 - MOVE GIT Dir: 'd:\PROJECT\git\dulwich' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:17.888000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\dulwich\' already exists\n')
2014-05-14T11:52:18.292000 - MOVE GIT Dir: 'd:\PROJECT\git\ECDRoom' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:18.318000 - MOVE GIT Dir: 'd:\PROJECT\git\eimage' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:18.348000 - MOVE GIT Dir: 'd:\PROJECT\git\Facebook.Python.SDK' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:18.367000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\Facebook.Python.SDK\' already exists\n')
2014-05-14T11:52:18.489000 - MOVE GIT Dir: 'd:\PROJECT\git\fast-export' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:18.518000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\fast-export\' already exists\n')
2014-05-14T11:52:18.637000 - MOVE GIT Dir: 'd:\PROJECT\git\FeedNotifier' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:18.641000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\FeedNotifier\' already exists\n')
2014-05-14T11:52:18.773000 - MOVE GIT Dir: 'd:\PROJECT\git\five-android' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:18.802000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\five-android\' already exists\n')
2014-05-14T11:52:19.505000 - MOVE GIT Dir: 'd:\PROJECT\git\five-server' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:19.530000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\five-server\' already exists\n')
2014-05-14T11:52:20.054000 - MOVE GIT Dir: 'd:\PROJECT\git\FixIndents' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.080000 - MOVE GIT Dir: 'd:\PROJECT\git\FreeSpace' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.395000 - MOVE GIT Dir: 'd:\PROJECT\git\ftpparsemodule' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.419000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\ftpparsemodule\' already exists\n')
2014-05-14T11:52:20.523000 - MOVE GIT Dir: 'd:\PROJECT\git\girocco' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.544000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\girocco\' already exists\n')
2014-05-14T11:52:20.590000 - MOVE GIT Dir: 'd:\PROJECT\git\git-cheetah' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.634000 - MOVE GIT Dir: 'd:\PROJECT\git\git-server' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.646000 - MOVE GIT Dir: 'd:\PROJECT\git\Gitalist' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.710000 - MOVE GIT Dir: 'd:\PROJECT\git\GitCandy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.743000 - MOVE GIT Dir: 'd:\PROJECT\git\GitPython' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.772000 - MOVE GIT Dir: 'd:\PROJECT\git\GitWeb' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.832000 - MOVE GIT Dir: 'd:\PROJECT\git\git_http_backend.py' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.897000 - MOVE GIT Dir: 'd:\PROJECT\git\gmail' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.949000 - MOVE GIT Dir: 'd:\PROJECT\git\gmail-sender' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:20.986000 - MOVE GIT Dir: 'd:\PROJECT\git\gmail-sender2' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:21.244000 - MOVE GIT Dir: 'd:\PROJECT\git\googleplay-api' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:21.367000 - MOVE GIT Dir: 'd:\PROJECT\git\icewrapmeraklog' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:21.710000 - MOVE GIT Dir: 'd:\PROJECT\git\imobis' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.033000 - MOVE GIT Dir: 'd:\PROJECT\git\libguestfs' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.265000 - MOVE GIT Dir: 'd:\PROJECT\git\libguestfs2' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.387000 - MOVE GIT Dir: 'd:\PROJECT\git\libuv' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.398000 - MOVE GIT Dir: 'd:\PROJECT\git\List_Com' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.409000 - MOVE GIT Dir: 'd:\PROJECT\git\List_HTML' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.454000 - MOVE GIT Dir: 'd:\PROJECT\git\mechanize' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.466000 - MOVE GIT Dir: 'd:\PROJECT\git\miglayoutfx2' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.530000 - MOVE GIT Dir: 'd:\PROJECT\git\mopidy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.566000 - MOVE GIT Dir: 'd:\PROJECT\git\mozbase' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.583000 - MOVE GIT Dir: 'd:\PROJECT\git\mva' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.638000 - MOVE GIT Dir: 'd:\PROJECT\git\MyGmail' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.732000 - MOVE GIT Dir: 'd:\PROJECT\git\noVNC' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.763000 - MOVE GIT Dir: 'd:\PROJECT\git\opt_parse_test' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:22.938000 - MOVE GIT Dir: 'd:\PROJECT\git\paramiko' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.006000 - MOVE GIT Dir: 'd:\PROJECT\git\php-git' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.251000 - MOVE GIT Dir: 'd:\PROJECT\git\pip' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.309000 - MOVE GIT Dir: 'd:\PROJECT\git\plowshare' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.322000 - MOVE GIT Dir: 'd:\PROJECT\git\py-mysql2pgsql' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.351000 - MOVE GIT Dir: 'd:\PROJECT\git\pyblue' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.408000 - MOVE GIT Dir: 'd:\PROJECT\git\PyBlueZ' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.482000 - MOVE GIT Dir: 'd:\PROJECT\git\pycmd-utils' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.550000 - MOVE GIT Dir: 'd:\PROJECT\git\pycrypto' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.587000 - MOVE GIT Dir: 'd:\PROJECT\git\pygreen' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.598000 - MOVE GIT Dir: 'd:\PROJECT\git\pygsm' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.609000 - MOVE GIT Dir: 'd:\PROJECT\git\pyhelp' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.671000 - MOVE GIT Dir: 'd:\PROJECT\git\pykka' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.733000 - MOVE GIT Dir: 'd:\PROJECT\git\pylogan' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:23.850000 - MOVE GIT Dir: 'd:\PROJECT\git\pylogsparser' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.089000 - MOVE GIT Dir: 'd:\PROJECT\git\PyMySQL' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.143000 - MOVE GIT Dir: 'd:\PROJECT\git\pyphantomjs' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.198000 - MOVE GIT Dir: 'd:\PROJECT\git\pypiserver' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.250000 - MOVE GIT Dir: 'd:\PROJECT\git\pyqtgraph' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.279000 - MOVE GIT Dir: 'd:\PROJECT\git\pyshorteners' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.290000 - MOVE GIT Dir: 'd:\PROJECT\git\python-decompiler' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.311000 - MOVE GIT Dir: 'd:\PROJECT\git\python-ecdsa' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.341000 - MOVE GIT Dir: 'd:\PROJECT\git\python.tmbundle' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.389000 - MOVE GIT Dir: 'd:\PROJECT\git\pytube' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.431000 - MOVE GIT Dir: 'd:\PROJECT\git\pyx' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.442000 - MOVE GIT Dir: 'd:\PROJECT\git\rarto' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.454000 - MOVE GIT Dir: 'd:\PROJECT\git\rbmail' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.465000 - MOVE GIT Dir: 'd:\PROJECT\git\redis' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.483000 - MOVE GIT Dir: 'd:\PROJECT\git\rento' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.513000 - MOVE GIT Dir: 'd:\PROJECT\git\rma' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.612000 - MOVE GIT Dir: 'd:\PROJECT\git\rtorrent' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.624000 - MOVE GIT Dir: 'd:\PROJECT\git\rubygsm' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.663000 - MOVE GIT Dir: 'd:\PROJECT\git\scp.py' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.715000 - MOVE GIT Dir: 'd:\PROJECT\git\scribe' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.727000 - MOVE GIT Dir: 'd:\PROJECT\git\searchinpath' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.779000 - MOVE GIT Dir: 'd:\PROJECT\git\sentry-whatsapp' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.827000 - MOVE GIT Dir: 'd:\PROJECT\git\sqbrowser' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.870000 - MOVE GIT Dir: 'd:\PROJECT\git\SuperParamiko' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.905000 - MOVE GIT Dir: 'd:\PROJECT\git\syslogging' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.944000 - MOVE GIT Dir: 'd:\PROJECT\git\SyslogProcessor' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.972000 - MOVE GIT Dir: 'd:\PROJECT\git\unknow' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:24.988000 - MOVE GIT Dir: 'd:\PROJECT\git\UnMask' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.014000 - MOVE GIT Dir: 'd:\PROJECT\git\vhostmaker' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.111000 - MOVE GIT Dir: 'd:\PROJECT\git\wazapp' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.126000 - MOVE GIT Dir: 'd:\PROJECT\git\webpy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.226000 - MOVE GIT Dir: 'd:\PROJECT\git\WebSocket-for-Python' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.270000 - MOVE GIT Dir: 'd:\PROJECT\git\websockify' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.282000 - MOVE GIT Dir: 'd:\PROJECT\git\Weld' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.348000 - MOVE GIT Dir: 'd:\PROJECT\git\winshell' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.361000 - MOVE GIT Dir: 'd:\PROJECT\git\yowsup' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.418000 - MOVE GIT Dir: 'd:\PROJECT\git\zipto' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.442000 - MOVE GIT Dir: 'd:\PROJECT\git-cheetah' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.449000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\git-cheetah\' already exists\n')
2014-05-14T11:52:25.502000 - MOVE GIT Dir: 'd:\PROJECT\Gitalist' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.508000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\Gitalist\' already exists\n')
2014-05-14T11:52:25.551000 - MOVE GIT Dir: 'd:\PROJECT\GitCandy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.557000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\GitCandy\' already exists\n')
2014-05-14T11:52:25.973000 - MOVE GIT Dir: 'd:\PROJECT\GitPython' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:25.979000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\GitPython\' already exists\n')
2014-05-14T11:52:26.045000 - MOVE GIT Dir: 'd:\PROJECT\GitWeb' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:26.052000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\GitWeb\' already exists\n')
2014-05-14T11:52:26.134000 - MOVE GIT Dir: 'd:\PROJECT\git_http_backend.py' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:26.141000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\git_http_backend.py\' already exists\n')
2014-05-14T11:52:26.394000 - MOVE GIT Dir: 'd:\PROJECT\gmail' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:26.418000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\gmail\' already exists\n')
2014-05-14T11:52:26.703000 - MOVE GIT Dir: 'd:\PROJECT\googleplay-api' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:52:26.729000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\googleplay-api\' already exists\n')
2014-05-14T11:52:27.542000 - MOVE HG Dir : 'd:\PROJECT\hg\7zme' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:27.609000 - MOVE HG Dir : 'd:\PROJECT\hg\7zme_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:27.623000 - MOVE HG Dir : 'd:\PROJECT\hg\albumplayer' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:28.054000 - MOVE HG Dir : 'd:\PROJECT\hg\aMTLog' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:28.157000 - MOVE HG Dir : 'd:\PROJECT\hg\androguard' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:30.159000 - MOVE HG Dir : 'd:\PROJECT\hg\androguard\elsim' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:30.842000 - MOVE HG Dir : 'd:\PROJECT\hg\android-notifier' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:30.996000 - MOVE HG Dir : 'd:\PROJECT\hg\apachce_configuration_backup' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:31.215000 - MOVE HG Dir : 'd:\PROJECT\hg\apache-wacher' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:31.366000 - MOVE HG Dir : 'd:\PROJECT\hg\apachelog_parser' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:31.578000 - MOVE HG Dir : 'd:\PROJECT\hg\Basemap' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:31.756000 - MOVE HG Dir : 'd:\PROJECT\hg\bookmark_center' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:32.610000 - MOVE HG Dir : 'd:\PROJECT\hg\bookmark_center(not used)' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:32.665000 - MOVE HG Dir : 'd:\PROJECT\hg\cantique_id' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:33.489000 - MOVE HG Dir : 'd:\PROJECT\hg\cd_dvdlist' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:33.771000 - MOVE HG Dir : 'd:\PROJECT\hg\ClueBzrServer-0.2' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:33.895000 - MOVE HG Dir : 'd:\PROJECT\hg\colorama' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:34.255000 - MOVE HG Dir : 'd:\PROJECT\hg\commandlineapp' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:34.394000 - MOVE HG Dir : 'd:\PROJECT\hg\conf' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:34.542000 - MOVE HG Dir : 'd:\PROJECT\hg\Control_servive_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:34.626000 - MOVE HG Dir : 'd:\PROJECT\hg\CopyMe_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:34.715000 - MOVE HG Dir : 'd:\PROJECT\hg\DBSearch_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:34.879000 - MOVE HG Dir : 'd:\PROJECT\hg\distribute' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:35.871000 - MOVE HG Dir : 'd:\PROJECT\hg\distutils2' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:36.484000 - MOVE HG Dir : 'd:\PROJECT\hg\django_hgadmin' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:36.627000 - MOVE HG Dir : 'd:\PROJECT\hg\evaluasiTIK-Python' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:37.586000 - MOVE HG Dir : 'd:\PROJECT\hg\evaluasiTIK_HG' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:38.371000 - MOVE HG Dir : 'd:\PROJECT\hg\evaluasiTIK_MAP' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:38.514000 - MOVE HG Dir : 'd:\PROJECT\hg\Fav-Log' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:38.677000 - MOVE HG Dir : 'd:\PROJECT\hg\fork-evaluasiTIK-Python' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:39.700000 - MOVE HG Dir : 'd:\PROJECT\hg\GBluz' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:39.813000 - MOVE HG Dir : 'd:\PROJECT\hg\helium_controll' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:39.888000 - MOVE HG Dir : 'd:\PROJECT\hg\hgaddin' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:39.982000 - MOVE HG Dir : 'd:\PROJECT\hg\hgcr-gui' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.051000 - MOVE HG Dir : 'd:\PROJECT\hg\hydra' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.151000 - MOVE HG Dir : 'd:\PROJECT\hg\iBee' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.402000 - MOVE HG Dir : 'd:\PROJECT\hg\imageedit' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.455000 - MOVE HG Dir : 'd:\PROJECT\hg\iTunes_Control' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.494000 - MOVE HG Dir : 'd:\PROJECT\hg\iTunes_on_WEB' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.550000 - MOVE HG Dir : 'd:\PROJECT\hg\kiwisyslog_web' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.756000 - MOVE HG Dir : 'd:\PROJECT\hg\Laptop_ID' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.861000 - MOVE HG Dir : 'd:\PROJECT\hg\link-maker' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:40.928000 - MOVE HG Dir : 'd:\PROJECT\hg\linker' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:41.009000 - MOVE HG Dir : 'd:\PROJECT\hg\listhdd' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:41.270000 - MOVE HG Dir : 'd:\PROJECT\hg\loadDLL_project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:41.313000 - MOVE HG Dir : 'd:\PROJECT\hg\login_for_etik' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:41.443000 - MOVE HG Dir : 'd:\PROJECT\hg\lucy' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:41.494000 - MOVE HG Dir : 'd:\PROJECT\hg\Lucy_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:41.592000 - MOVE HG Dir : 'd:\PROJECT\hg\mercurial' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:42.663000 - MOVE HG Dir : 'd:\PROJECT\hg\messaging' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:42.738000 - MOVE HG Dir : 'd:\PROJECT\hg\moztalos' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.013000 - MOVE HG Dir : 'd:\PROJECT\hg\MyFilebank_Project' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.078000 - MOVE HG Dir : 'd:\PROJECT\hg\mylisthdd' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.188000 - MOVE HG Dir : 'd:\PROJECT\hg\my_python_tutorial_script' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.245000 - MOVE HG Dir : 'd:\PROJECT\hg\nod32_log_web' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.451000 - MOVE HG Dir : 'd:\PROJECT\hg\Notify' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.576000 - MOVE HG Dir : 'd:\PROJECT\hg\notify_server' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.636000 - MOVE HG Dir : 'd:\PROJECT\hg\packcrack' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.733000 - MOVE HG Dir : 'd:\PROJECT\hg\php-in-python' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:43.826000 - MOVE HG Dir : 'd:\PROJECT\hg\Process_list' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.026000 - MOVE HG Dir : 'd:\PROJECT\hg\Project_Bluetooth' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.091000 - MOVE HG Dir : 'd:\PROJECT\hg\Project_LISTHDD' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.146000 - MOVE HG Dir : 'd:\PROJECT\hg\Project_log' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.216000 - MOVE HG Dir : 'd:\PROJECT\hg\PyDot_examples' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.252000 - MOVE HG Dir : 'd:\PROJECT\hg\pyevaluasitik' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.309000 - MOVE HG Dir : 'd:\PROJECT\hg\pygments' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.560000 - MOVE HG Dir : 'd:\PROJECT\hg\pygraphviz-1.1' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:44.727000 - MOVE HG Dir : 'd:\PROJECT\hg\pyload' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.074000 - MOVE HG Dir : 'd:\PROJECT\hg\pymail' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.127000 - MOVE HG Dir : 'd:\PROJECT\hg\pymailrw' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.184000 - MOVE HG Dir : 'd:\PROJECT\hg\pyobex' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.293000 - MOVE HG Dir : 'd:\PROJECT\hg\pypowercontrol' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.596000 - MOVE HG Dir : 'd:\PROJECT\hg\PySNMP' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.682000 - MOVE HG Dir : 'd:\PROJECT\hg\PySock' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.723000 - MOVE HG Dir : 'd:\PROJECT\hg\pysyslog2db' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.903000 - MOVE HG Dir : 'd:\PROJECT\hg\pysyslog2db2' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:46.990000 - MOVE HG Dir : 'd:\PROJECT\hg\pysyslog2db_first' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:47.103000 - MOVE HG Dir : 'd:\PROJECT\hg\python-processes' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:47.212000 - MOVE HG Dir : 'd:\PROJECT\hg\python-rock-solid-tools' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:47.526000 - MOVE HG Dir : 'd:\PROJECT\hg\pyx_first' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:48.747000 - MOVE HG Dir : 'd:\PROJECT\hg\pyx_hg' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:50.471000 - MOVE HG Dir : 'd:\PROJECT\hg\QStyle' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:50.552000 - MOVE HG Dir : 'd:\PROJECT\hg\ReadSearch' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:50.655000 - MOVE HG Dir : 'd:\PROJECT\hg\RegCleaner' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:50.817000 - MOVE HG Dir : 'd:\PROJECT\hg\roundup-code' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:52.571000 - MOVE HG Dir : 'd:\PROJECT\hg\rss' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:52.714000 - MOVE HG Dir : 'd:\PROJECT\hg\scpclient' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:52.755000 - MOVE HG Dir : 'd:\PROJECT\hg\service_manipulation' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:52.841000 - MOVE HG Dir : 'd:\PROJECT\hg\simpel' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:57.419000 - MOVE HG Dir : 'd:\PROJECT\hg\simpel-code' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:52:57.857000 - MOVE HG Dir : 'd:\PROJECT\hg\simpel-for-natuna' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:01.770000 - MOVE HG Dir : 'd:\PROJECT\hg\SIMPEN' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:05.231000 - MOVE HG Dir : 'd:\PROJECT\hg\simpledns_controll' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:05.351000 - MOVE HG Dir : 'd:\PROJECT\hg\smart-notify' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:05.409000 - MOVE HG Dir : 'd:\PROJECT\hg\startmenu' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:05.496000 - MOVE HG Dir : 'd:\PROJECT\hg\syslogwacher_web' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:05.809000 - MOVE HG Dir : 'd:\PROJECT\hg\syslog_center' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:06.391000 - MOVE HG Dir : 'd:\PROJECT\hg\syslog_center_server' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:06.952000 - MOVE HG Dir : 'd:\PROJECT\hg\TotalcmdLog' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:07.229000 - MOVE HG Dir : 'd:\PROJECT\hg\totalcmdlog_web' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:07.782000 - MOVE HG Dir : 'd:\PROJECT\hg\vhost_control' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:07.903000 - MOVE HG Dir : 'd:\PROJECT\hg\WAMP_Control' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:07.982000 - MOVE HG Dir : 'd:\PROJECT\hg\wb2pdf' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:08.478000 - MOVE HG Dir : 'd:\PROJECT\hg\whoosh' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:08.582000 - MOVE HG Dir : 'd:\PROJECT\hg\wit-0.0.4' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:08.941000 - MOVE HG Dir : 'd:\PROJECT\hg\woodenbrick-gtkPopupNotify' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:09.028000 - MOVE HG Dir : 'd:\PROJECT\hg\wordcase' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:09.103000 - MOVE HG Dir : 'd:\PROJECT\hg\wphp' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:09.868000 - MOVE HG Dir : 'd:\PROJECT\hg\XAMPP_Control' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:09.965000 - MOVE HG Dir : 'd:\PROJECT\hg\y510' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:10.036000 - MOVE GIT Dir: 'd:\PROJECT\imobis' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:10.072000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\imobis\' already exists\n')
2014-05-14T11:53:10.215000 - MOVE GIT Dir: 'd:\PROJECT\libguestfs' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:10.253000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\libguestfs\' already exists\n')
2014-05-14T11:53:11.329000 - MOVE GIT Dir: 'd:\PROJECT\libguestfs2' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:11.361000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\libguestfs2\' already exists\n')
2014-05-14T11:53:12.032000 - MOVE GIT Dir: 'd:\PROJECT\libuv' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:12.084000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\libuv\' already exists\n')
2014-05-14T11:53:12.190000 - MOVE GIT Dir: 'd:\PROJECT\mechanize' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:12.220000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\mechanize\' already exists\n')
2014-05-14T11:53:12.373000 - MOVE GIT Dir: 'd:\PROJECT\miglayoutfx2' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:12.434000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\miglayoutfx2\' already exists\n')
2014-05-14T11:53:12.518000 - MOVE GIT Dir: 'd:\PROJECT\mopidy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:12.563000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\mopidy\' already exists\n')
2014-05-14T11:53:12.736000 - MOVE GIT Dir: 'd:\PROJECT\mozbase' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:12.770000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\mozbase\' already exists\n')
2014-05-14T11:53:12.817000 - MOVE GIT Dir: 'd:\PROJECT\noVNC' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:12.849000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\noVNC\' already exists\n')
2014-05-14T11:53:13.064000 - MOVE GIT Dir: 'd:\PROJECT\paramiko' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:13.099000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\paramiko\' already exists\n')
2014-05-14T11:53:14.124000 - MOVE GIT Dir: 'd:\PROJECT\pip' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:14.159000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pip\' already exists\n')
2014-05-14T11:53:16.335000 - MOVE GIT Dir: 'd:\PROJECT\plowshare' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:16.413000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\plowshare\' already exists\n')
2014-05-14T11:53:16.869000 - MOVE GIT Dir: 'd:\PROJECT\pyblue' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:16.901000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pyblue\' already exists\n')
2014-05-14T11:53:16.964000 - MOVE GIT Dir: 'd:\PROJECT\PyBlueZ' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:16.993000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\PyBlueZ\' already exists\n')
2014-05-14T11:53:17.363000 - MOVE GIT Dir: 'd:\PROJECT\pycrypto' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:17.491000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pycrypto\' already exists\n')
2014-05-14T11:53:18.333000 - MOVE GIT Dir: 'd:\PROJECT\pygreen' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:18.393000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pygreen\' already exists\n')
2014-05-14T11:53:18.615000 - MOVE GIT Dir: 'd:\PROJECT\pygsm' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:18.665000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pygsm\' already exists\n')
2014-05-14T11:53:18.789000 - MOVE GIT Dir: 'd:\PROJECT\pykka' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:18.823000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pykka\' already exists\n')
2014-05-14T11:53:19.257000 - MOVE GIT Dir: 'd:\PROJECT\pylogan' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:19.301000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pylogan\' already exists\n')
2014-05-14T11:53:19.635000 - MOVE GIT Dir: 'd:\PROJECT\pylogsparser' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:19.707000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pylogsparser\' already exists\n')
2014-05-14T11:53:19.869000 - MOVE GIT Dir: 'd:\PROJECT\PyMySQL' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:19.900000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\PyMySQL\' already exists\n')
2014-05-14T11:53:19.988000 - MOVE GIT Dir: 'd:\PROJECT\pyphantomjs' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:20.020000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pyphantomjs\' already exists\n')
2014-05-14T11:53:20.309000 - MOVE GIT Dir: 'd:\PROJECT\pypiserver' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:20.340000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pypiserver\' already exists\n')
2014-05-14T11:53:20.468000 - MOVE GIT Dir: 'd:\PROJECT\pyqtgraph' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:20.509000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pyqtgraph\' already exists\n')
2014-05-14T11:53:20.900000 - MOVE GIT Dir: 'd:\PROJECT\pyshorteners' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:20.932000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pyshorteners\' already exists\n')
2014-05-14T11:53:20.973000 - MOVE HG Dir : 'd:\PROJECT\pysyslog2db' --> 'd:\PROJECT\BCK\hg'
2014-05-14T11:53:21.118000 - MOVE GIT Dir: 'd:\PROJECT\python-ecdsa' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:21.149000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\python-ecdsa\' already exists\n')
2014-05-14T11:53:21.179000 - MOVE GIT Dir: 'd:\PROJECT\python.tmbundle' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:21.212000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\python.tmbundle\' already exists\n')
2014-05-14T11:53:21.284000 - MOVE GIT Dir: 'd:\PROJECT\pytube' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:21.317000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\pytube\' already exists\n')
2014-05-14T11:53:21.401000 - MOVE GIT Dir: 'd:\PROJECT\rtorrent' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:21.432000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\rtorrent\' already exists\n')
2014-05-14T11:53:21.578000 - MOVE GIT Dir: 'd:\PROJECT\rubygsm' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:21.611000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\rubygsm\' already exists\n')
2014-05-14T11:53:21.683000 - MOVE GIT Dir: 'd:\PROJECT\scribe' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:21.732000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\scribe\' already exists\n')
2014-05-14T11:53:21.992000 - MOVE GIT Dir: 'd:\PROJECT\sqbrowser' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:22.032000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\sqbrowser\' already exists\n')
2014-05-14T11:53:22.078000 - MOVE SVN Dir: 'd:\PROJECT\svn\copysprite-read-only' --> 'd:\PROJECT\BCK\svn'
2014-05-14T11:53:22.141000 - MOVE SVN Dir: 'd:\PROJECT\svn\htql' --> 'd:\PROJECT\BCK\svn'
2014-05-14T11:53:22.243000 - MOVE SVN Dir: 'd:\PROJECT\svn\htql-read-only' --> 'd:\PROJECT\BCK\svn'
2014-05-14T11:53:22.306000 - MOVE SVN Dir: 'd:\PROJECT\svn\portablephpserver-read-only' --> 'd:\PROJECT\BCK\svn'
2014-05-14T11:53:22.428000 - MOVE SVN Dir: 'd:\PROJECT\svn\protobuf-read-only' --> 'd:\PROJECT\BCK\svn'
2014-05-14T11:53:22.485000 - MOVE SVN Dir: 'd:\PROJECT\svn\pyorg-read-only' --> 'd:\PROJECT\BCK\svn'
2014-05-14T11:53:22.541000 - MOVE GIT Dir: 'd:\PROJECT\SyslogProcessor' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:22.570000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\SyslogProcessor\' already exists\n')
2014-05-14T11:53:22.841000 - MOVE GIT Dir: 'd:\PROJECT\wazapp' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:22.875000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\wazapp\' already exists\n')
2014-05-14T11:53:24.470000 - MOVE GIT Dir: 'd:\PROJECT\webpy' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:24.501000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\webpy\' already exists\n')
2014-05-14T11:53:24.789000 - MOVE GIT Dir: 'd:\PROJECT\WebSocket-for-Python' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:24.825000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\WebSocket-for-Python\' already exists\n')
2014-05-14T11:53:25.826000 - MOVE GIT Dir: 'd:\PROJECT\websockify' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:25.862000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\websockify\' already exists\n')
2014-05-14T11:53:25.929000 - MOVE GIT Dir: 'd:\PROJECT\Weld' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:25.967000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\Weld\' already exists\n')
2014-05-14T11:53:26.085000 - MOVE GIT Dir: 'd:\PROJECT\winshell' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:26.119000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\winshell\' already exists\n')
2014-05-14T11:53:26.165000 - MOVE GIT Dir: 'd:\PROJECT\yowsup' --> 'd:\PROJECT\BCK\git'
2014-05-14T11:53:26.198000 - ('Error:', 'Traceback (most recent call last):\n  File "C:\\pyx\\repomgmt.py", line 237, in analisys\n    shutil.move(os.path.dirname(git), dst_git)\n  File "C:\\Python27\\lib\\shutil.py", line 291, in move\n    raise Error, "Destination path \'%s\' already exists" % real_dst\nError: Destination path \'d:\\PROJECT\\BCK\\git\\yowsup\' already exists\n')
